### PR TITLE
ShipmentView: Implement History Timeline (First Iteration)

### PR DESCRIPTION
**Pull request by vahidbazzaz**
The PR is the first iteration of the shipment history timeline implementation, it compiles the history entries for shipments and updates the history tab interface, but the items are not grouped and further improvements are needed.

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[2.0 Implement History Timeline in the ShipmentView](https://trello.com/c/cU4glRB0)

File: react/src/views/Transfers/ShipmentView/components/ShipmentHistory.tsx [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
+import { Box, Text, UnorderedList } from "@chakra-ui/react";
+import { IGroupedHistoryEntry, IShipmentHistory, ShipmentActionEvent } from "./ShipmentTabs";
 import TimelineEntry from "./TimelineEntry";
 
 export interface IShipmentHistoryProps {
   histories: IGroupedHistoryEntry[];
 }
 
 function ShipmentHistory({ histories }: IShipmentHistoryProps) {
+  const changesLabel = (histoty: IShipmentHistory): string => {
```
Comment (pylipp) [inline react/src/views/Transfers/ShipmentView/components/ShipmentHistory.tsx:10]: typo: history

**Review by pylipp — APPROVED** [review]
:heavy_check_mark: functional review. The list of history entries is compiled and is updated when I perform actions (remove box, send shipment, mark shipment as lost). Only two minor remarks. Thank you :blush:

**Comment by aerinsol** [comment]
@vahidbazzaz , Shipment state changes (marking as lost or sent) doesn't work for me on staging.

**Comment by pylipp** [comment]
@aerinsol The PR was merged into [feature/box-transfer-shipment-view-fix](https://github.com/boxwise/boxtribute/tree/feature/box-transfer-shipment-view-fix), not master, hence it's not effective in v2-staging.

**Comment by aerinsol** [comment]
Thanks for the clarification @pylipp . I will merge the other one since it's passed your functional review to test.

**Comment by aerinsol** [comment]
@HaGuesto this is one PR. @vahidbazzaz please update your original comment per what we agreed in Slack for non-master bases.

**Pull request merged**